### PR TITLE
Add l3gd20 gyroscope driver

### PR DESCRIPTION
**Pull request by soypat**
Gyroscope driver with example.

### CODE REVIEW

File: examples/l3gd20/main.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		}
+		x, y, z = gyro.AngularVelocity()
+		println(x, y, z)
+		time.Sleep(500 * time.Millisecond)
+	}
+}
+
+// ScanI2CDev finds I2C devices on the bus and rreturns them inside
+// a slice. If slice is nil then no devices were found.
+func ScanI2CDev(bus *machine.I2C) (addrs []uint8) {
```
Comment (deadprogram) [inline examples/l3gd20/main.go]: Cool function, but is not being called and also probably doesn't belong in this example.

Separate note, I worked on this a little last year, but just found the snippet and made a gist now: https://gist.github.com/deadprogram/6e1f3ecdb79cf223db2fdaeb4534bfe6

Comment (deadprogram) [reply examples/l3gd20/main.go]: So perhaps this function can be removed from here?

Comment (soypat) [reply examples/l3gd20/main.go]: Yes! I'll fix this ASAP

Comment (soypat) [reply examples/l3gd20/main.go]: Done!

**Comment by deadprogram** [comment]
Thanks @soypat now merging.

**Pull request merged**